proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for get real system time during TLS handshake.
+
+
+## Motivation
+
+Now SDL Core does not have access to the real system time so  need an interface to the system in order to request it when it is required 
+
+
+## Proposed solution
+
+
+The solution proposes adding new RPC between SDL and HMI.  GetSystemTime gives possibility to get the real system time status by introducing new API.
+
+SDL Core must send this GetSystemTime RPC to HMI in case of Policy Table Update sequence - in case there is no certificate on the system.  Than system time will be required to finish Security TLS Handshake.

```

Comment:
@AStasiuk 
My previous comment basically mention policy update just to make it more clear for you but I don't think it has to be mentioned in proposal as policy update has nothing to do with system time

Reply:
@dev-gh Fixed in https://github.com/AStasiuk/sdl_evolution/pull/1/files#diff-8c90e61f2f7752dccc9f3a2e657dd2aa